grakn-graql/src/main/java/ai/grakn/graql/internal/analytics/MedianVertexProgram.java

```diff
@@ -129,7 +129,7 @@ public void storeState(final Configuration configuration) {
     public void loadState(final Graph graph, final Configuration configuration) {
         super.loadState(graph, configuration);
         configuration.subset(RESOURCE_TYPE).getKeys().forEachRemaining(key ->
-                statisticsResourceLabelIds.add(LabelId.of(configuration.getInt(RESOURCE_TYPE + "." + key))));
+                statisticsResourceLabelIds.add((LabelId) configuration.getProperty(RESOURCE_TYPE + "." + key)));
```

Comment:
It can catch the failure of spark computer in standalone mode, but on a real cluster I guess it's more difficult to test this